lib/collection/property-base.js

```diff
@@ -1,5 +1,6 @@
 var _ = require('../util').lodash,
     Description = require('./description').Description,
+    PropertyList = require('./property-list').PropertyList,
```

Comment:
Actually it is only needed because we internally have plural names for lists, but the collection format has singular names. [headers vs header, items vs item, etc]